server/src/domain/job/job.service.ts

```diff
@@ -163,9 +164,15 @@ export class JobService {
         await this.jobRepository.queue({ name: JobName.CLASSIFY_IMAGE, data: item.data });
         await this.jobRepository.queue({ name: JobName.ENCODE_CLIP, data: item.data });
         await this.jobRepository.queue({ name: JobName.RECOGNIZE_FACES, data: item.data });
+        if (item.data.source !== 'upload') {
```

Comment:
What is the reason for this check here?

Reply:
Videos are currently only automatically queued for transcoding on upload, not when any other job is run. This check is to maintain that behavior. The asset check in this section also seems specific to uploading.